clj-refactor.el

```diff
@@ -3254,6 +3254,17 @@ if REMOVE-PACKAGE_VERSION is t get rid of the (package: 20150828.1048) suffix."
         (replace-regexp-in-string " (.*)" "" version)
       version)))
 
+(defcustom get-cljr-version (lambda ()
+                              (cljr--version t))
+  "A no-args function that should return the refactor-nrepl version to be used.
+
+You can customize this in order to try out new releases.
+
+If customizing it, you most likely should `(setq cljr-suppress-middleware-warnings nil)' too,
+for avoiding a warning that would be irrelevant for this case."
+  :group 'cljr
+  :type 'function)
```

Comment:
I'd also add a `:package-version` property here, so it's easy to spot when this was added.